processing/src/main/java/org/apache/druid/segment/column/ColumnCapabilities.java

```diff
@@ -57,6 +50,21 @@ public boolean isTrue()
       return this == TRUE;
     }
 
+    public boolean isMaybeTrue()
+    {
+      return isTrue() || isUnknown();
+    }
+
+    public boolean isUnknown()
+    {
+      return this == UNKNOWN;
+    }
+
+    public Capable complete(boolean convertUnknownToTrue)
```

Comment:
I don't think it needs another enum, if anything it could just accept a `Capable` to use for the default value, though I should enforce that it cannot be unknown. If i rename to `coerceIfUnknown` or something maybe it would also be more obvious what the purpose is.